smartsim/_core/_install/buildenv.py

```diff
@@ -58,24 +61,28 @@ class SetupError(Exception):
     See setup.py for more
     """
 
-    pass
```

Comment:
We don't need the pass as an explicit declaration that the class is a name-only overload?

Reply:
If we didn't have the comment, the pass would be required.